build_tools/release_tools.py

```diff
@@ -61,21 +61,45 @@ def get_asset_sha(org_name, repo_name, tag, workdir):
     print("sha: {v}".format(v=sha))
     return sha
 
-def prep_conda_env(to_do_conda_clean=False):
-    if to_do_conda_clean:
-        cmd = "conda clean --all"
-        ret = run_cmd(cmd, join_stderr, shell_cmd, verbose, workdir)
+def prep_conda_env(conda_activate, conda_rc, conda_env, extra_channels, to_do_conda_clean=False, **kwargs):
```

Comment:
I am not too familiar with **kwargs. Do you still need 'conda_activate', 'conda_rc', 'conda_env' and 'extra_channels'?

Reply:
By having those we're unpacking those specific values from `**kwargs` otherwise we'd be having to use `kwargs['conda_rc']` at ever point we use the `conda_rc` variable. kwargs is acting as a bag for all the parameters we don't need.

Reply:
ah..I see. ok. thanks.